### PR TITLE
Add media query helper

### PR DESCRIPTION
**Pull request by notwaldorf**
+ forgot to add seo-metadata section to the readme so did that too

### CODE REVIEW

File: demo/components/my-app.js [hunk 1]
```diff
@@ -87,10 +94,12 @@ class MyApp extends connect(store)(HTMLElement) {
     });
 
     installRouter(() => store.dispatch(navigate(window.location)));
-
     installOfflineWatcher((offline) => {
       this._offline.textContent = offline ? ' offline' : 'online';
     });
+    installMediaQueryWatcher(`(min-width: 700px)`, (matches) => {
```
Comment (frankiefu) [inline demo/components/my-app.js]: change to `600px` to match the comment above, or vice versa.

Comment (notwaldorf) [reply demo/components/my-app.js]: 🤦‍♂️

**Comment by abdonrd** [comment]
@notwaldorf I think you forget to push the `media-query.js` 😅

**Comment by notwaldorf** [comment]
🙀😳😭

**Pull request merged**